.pipelines/vhd-builder-windows.yaml

```diff
@@ -0,0 +1,29 @@
+trigger: none
+
+jobs:
+- job: build_vhd_windows
+  timeoutInMinutes: 120
+  strategy:
+    maxParallel: 0
+  pool:
+    name: Hosted Ubuntu 1604
+  steps:
+  - script: |
+      docker run --rm \
+      -v ${PWD}:/go/src/github.com/Azure/aks-engine \
+      -w /go/src/github.com/Azure/aks-engine \
+      -e CLIENT_ID=${CLIENT_ID} \
+      -e CLIENT_SECRET="$(CLIENT_SECRET)" \
+      -e TENANT_ID=${TENANT_ID} \
+      -e AZURE_RESOURCE_GROUP_NAME=${AZURE_RESOURCE_GROUP_NAME} \
+      -e AZURE_LOCATION=${AZURE_LOCATION} \
+      -e AZURE_VM_SIZE=${AZURE_VM_SIZE} \
+      -e GIT_VERSION=$(Build.SourceVersion) \
+      -e BUILD_ID=$(Build.BuildId) \
+      -e BUILD_NUMBER=$(Build.BuildNumber) \
+      ${DEIS_GO_DEV_IMAGE} make run-packer-windows
+    displayName: Building windows VHD
+  - task: PublishPipelineArtifact@0
+    inputs:
+      artifactName: 'vhd-release-notes'
+      targetPath: 'release-notes.txt'

```

Comment:
is there any reason we should add a timestamp or git hash to this filename? Not sure if we need to archive them into a single folder someplace

Reply:
The Linux pipeline uses a constant file name for the release notes as well. I'm not sure if those get archived or anything.
@CecileRobertMichon can you comment on this?

Reply:
The devops job doesn't really need a timestamp because there's always just one artifact with that name per job (and we never rebuild a job, if anything we trigger a new build). When you download the artifact from devops it gets renames anyways so when we add the file to the repo we have to rename it no matter what. We archive the release notes of published images here: https://github.com/Azure/aks-engine/tree/master/releases/vhd-notes (the name of the file is sku_version of the AKS offer image).